index.html

```diff
@@ -137,30 +152,37 @@ <h2>Multi-Post Stories</h2>
             </ul>
           </div>
           <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+            A smart assistant to make driving more safe, efficient, and fun by
+            unlocking your most expensive computer: your car.
           </p>
           <ul class="project-tech">
             <li>html</li>
+            <li>Ruby on rails</li>
             <li>css</li>
             <li>javascript</li>
           </ul>
-   
-            <a class="project-button" href="">See Project</a>
-          
+
+          <a class="project-button" href="">See Project</a>
         </div>
+        <img
+          class="desktop"
+          src="images/Snapshoot Portfolio4.svg"
+          alt="Project photo"
+        />
       </div>
     </section>
 
     <section class="about-section">
```

Comment:
[ ] kindly adjust the work section to be similar to the Figma design 👀, it should take 40% of the container's width, and 4 the text should be 4 to 5 lines.
Your design : 
![histext](https://user-images.githubusercontent.com/62156607/193083178-fb7f5d1b-fc12-4323-add8-532e3534fc4d.PNG)

Figma design : 
![destext](https://user-images.githubusercontent.com/62156607/193083214-65ca9399-8296-4426-a780-b86ec348fbb4.PNG)